### DISCUSSION TITLE
Docker Setup Wizard Certificate Password Protected

### DISCUSSION THREAD

**Discussion by CyberCyclone** [ravendb/ravendb#13051] [Q&A] [answered]
Hi,

I'm trying to setup a cluster with Docker however after it downloads the certificate in the Setup Wizard I run into this:

<img width="619" alt="image" src="https://user-images.githubusercontent.com/4373888/141923212-32bb173d-43f7-4926-a0ea-66180e850d7f.png">

```
This server requires client certificate for authentication, but none was provided by the client. Did you forget to install the certificate?

Your certificate store may be cached by the browser. Create a new private browsing tab, which will not cache any certificates. (Ctrl+Shift+N in Chrome, Ctrl+Shift+P in Firefox)
```

I've mounted the settings file with `/home/cgibson/RavenDb/settings.json:/opt/RavenDB/Server/settings.json`

and certificate ` - /home/cgibson/RavenDb/cluster.server.certificate.adatools-io.pfx:/opt/RavenDB/Server/cluster.server.certificate.adatools-io.pfx
`

I've been able to load `admin.client.certificate.adatools-io.crt` into MacOS however the `admin.client.certificate.adatools-io.pfx` file is password protected. The setup wizard never asked for a password and never gave one. I've tried this twice now with no luck and there's no documentation around it - any documentation simply says to select it.

Any help would be appreciated, thanks!

**Comment by ayende** [comment, marked as answer]
The `crt` file is only the public key portion of the key. You need to register the `key` file as well.
There is no password defined for the `pfx` from the client by default, there is another `pfx` file that is password protected (and the password is in the zip).

See if you can use a blank password or use the `pfx` that is password protected.

**Comment by CyberCyclone** [comment]
I was able to get `admin.client...pfx` added to Firefox by just hitting `enter` with an empty password as you suggested. MacOS doesn't allow you to do that when adding to the keychain, so this definitely doesn't work with Safari (as the dialog box rightly points out when first going to the site in Safari). 

Thanks!
